modules/dns/variables.tf

```diff
@@ -25,6 +25,41 @@ variable "client_networks" {
   nullable    = false
 }
 
+variable "create_recordsets" {
```

Comment:
Can we implement this via an `update_only = optional(bool, false)` in the `recordsets` variable?

Reply:
You probably mean `create_only` but let me try.

Reply:
yep `create_only` sorry :)

Reply:
Added the `create_only` flag.